src/map/mob.cpp

```diff
@@ -3854,7 +3854,7 @@ int mobskill_use(struct mob_data *md, t_tick tick, int event)
 		if ( ms[i]->msg_id ){ //Display color message [SnakeDrak]
 			std::shared_ptr<s_mob_chat> mc = mob_chat_db.find(ms[i]->msg_id);
 
-			if (mc) {
+			if (mc && mc->msg.length() > 0) {	// skip if empty
```

Comment:
Wouldnt it be better to remove the entry from the database, if it is empty?
Also check out #5974.
This way users can remove an entry by "Remove: <message id>" if it is merged.

Reply:
I misread your PR sorry. I will revert my last change